recipes-ni/nisdbootconfig/files/nisdbootconfig

```diff
@@ -2,19 +2,15 @@
 set -e
 
 # check if target is NOT using ubifs (mtd)
-if ! cat /proc/mtd | grep -qs mtd ; then
+if ! grep -qs mtd /proc/mtd ; then
```

Comment:
I've also amended this line as per @gratian 's comment over at the review-board that discussed about the redundancy of using "cat" in this case. Refer to https://review-board.natinst.com/r/356932/.